3rdParty/StormLib/src/SFileOpenArchive.cpp

```diff
@@ -201,7 +201,6 @@ LCID WINAPI SFileSetLocale(LCID lcNewLocale)
 //   dwPriority - When SFileOpenFileEx called, this contains the search priority for searched archives
```

Comment:
The changes in this file look like formatting-only changes

Reply:
Some of those were leftovers from when I was debugging. I'll remove them.